cli/src/bin/ulid.rs

```diff
@@ -0,0 +1,81 @@
+#[macro_use]
+extern crate clap;
+
+use clap::{Arg, Values};
+use std::io::{self, Write};
+use ulid::EncodingError;
+use ulid::Ulid;
+
+const ARG_COUNT: &str = "count";
+const ARG_ULID: &str = "<ULID>";
+
+fn main() -> Result<(), String> {
+    let matches = app_from_crate!("\n")
```

Comment:
I prefer using [structopt](https://crates.io/crates/structopt), but if you don't want to change this, that's fine.

Reply:
I didn't know structopt (well, I 🌟 the repository but I forgot). It looks very interesting, especially I was also struggling about the validation. Let me try to get it to work.